doc/api/n-api.md

```diff
@@ -4803,8 +4803,8 @@ a specific `napi_env`.
 ### napi_get_uv_event_loop
 <!-- YAML
 added:
-  - v8.10.0
-  - v9.3.0
+  * v8.10.0
+  * v9.3.0
```

Comment:
This as well is YAML embedded in an HTML comment, not a markdown unordered list.